srcpkgs/haskell-language-server/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'haskell-language-server'
+pkgname=haskell-language-server
+version=1.0.0
+revision=1
+archs="x86_64"
+build_style="haskell-stack"
+make_build_args="--stack-yaml stack-8.8.4.yaml"
+makedepends="ncurses-devel ncurses-libtinfo-devel icu-devel zlib-devel"
+short_desc="Haskell Language Server: Integration of ghcide and haskell-ide-engine"
+maintainer="Wayne Van Son <waynevanson@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/haskell/haskell-language-server"
+distfiles="https://github.com/haskell/haskell-language-server/archive/${version}.tar.gz"
+checksum="14e28d6621d029f027fae44bc4a4ef62c869dab24ff01b88a2e51e6679cbff6c"
+nopie_files="
+/usr/bin/haskell-language-server
+/usr/bin/haskell-language-server-wrapper
+/usr/bin/ghcide-bench
+/usr/bin/ghcide
+/usr/bin/ghcide-test-preprocessor
+/usr/bin/test-server
+"
+nocross="Haskell package 'Network' builds the hosts architecture, not the target's"
+
+pre_build() {
+	# fixes /usr/bin/ld: cannot find -lncursesw
+	#
+	# currently not required since cross compilation is off,
+	# but leaving here for the future.
+	if [  "$XBPS_TARGET_MACHINE" != "x86_64" ] && [ ! -f "/lib/libncursesw.so" ]; then
+		 ln -s /lib/libncursesw.so.6 /lib/libncursesw.so
```

Comment:
Leaving changes in masterdir is not best idea.

Reply:
Since we're only building on x86_64 so far, this line isn't necessary.

I would like to keep it here as a reference. Would I be able to comment it out as a resolution?

Reply:
I prefer to have it removed. Not only modifying masterdir and implying that `[ "$XBPS_TARGET_MACHINE" != "x86_64" ]` equals `[ "$CROSS_BUILD" ]` is wrong, also it has good chance to be unnecessary once we can cross compile with stack.